src/app/shared/chips/chips.component.ts

```diff
@@ -79,16 +85,34 @@ export class ChipsComponent implements OnChanges {
 
   showTooltip(tooltip: NgbTooltip, index, field?) {
     tooltip.close();
-    const item = this.chips.getChipByIndex(index);
-    if (!item.editMode && this.dragged === -1) {
+    const chipsItem = this.chips.getChipByIndex(index);
+    const textToDisplay: string[] = [];
+    if (!chipsItem.editMode && this.dragged === -1) {
       if (field) {
-        this.tipText = (isObject(item.item[field])) ? item.item[field].display : item.item[field];
+        if (isObject(chipsItem.item[field])) {
+          textToDisplay.push(chipsItem.item[field].display);
+          if (chipsItem.item[field].hasOtherInformation()) {
+            Object.keys(chipsItem.item[field].otherInformation)
+              .forEach((otherField) => {
+                this.translate.get('form.other-information.' + otherField)
+                  .subscribe((label) => {
+                    textToDisplay.push(label + ': ' + chipsItem.item[field].otherInformation[otherField]);
+                  })
+            })
+          }
+        } else {
+          textToDisplay.push(chipsItem.item[field]);
+        }
       } else {
-        this.tipText = item.display;
+        textToDisplay.push(chipsItem.display);
       }
 
       this.cdr.detectChanges();
-      tooltip.open();
+      if (!chipsItem.hasIcons() || (chipsItem.hasIcons() && !chipsItem.hasVisibleIcons()) || field) {
```

Comment:
I don't think the second `chipsItem.hasIcons()` is necessary. If `!chipsItem.hasIcons()` is false (and therefore we move to the second part of the OR), `chipsItem.hasIcons()` must be true